src/backend/marsha/core/serializers.py

```diff
@@ -221,7 +221,7 @@ def get_source_url(self, obj):
             None if the timed text track is still not uploaded to S3 with success.
 
         """
-        if obj.uploaded_on:
+        if obj.uploaded_on and obj.extension:
```

Comment:
So if it has no extension you return no source url and display no download link? Why not computing the source url without extension? Do we know what are are the cases when there is no extension?